src/common/TrackedOp.h

```diff
@@ -15,17 +15,20 @@
 #define TRACKEDREQUEST_H_
 #include <sstream>
 #include <stdint.h>
-#include <include/utime.h>
+#include <boost/intrusive/list.hpp>
+#include <atomic>
+
+#include "include/utime.h"
 #include "common/Mutex.h"
 #include "common/histogram.h"
-#include "include/xlist.h"
 #include "msg/Message.h"
 #include "include/memory.h"
 #include "common/RWLock.h"
-#include <atomic>
+
+#define OPTRACKER_PREALLOC_EVENTS 20
```

Comment:
Any reason not to make this a config value? How did you choose 20? (That seems awfully small.)